docs/OpenCLCOnVulkan.md

```diff
@@ -757,7 +757,13 @@ Instead, those types are mapped to 32-bit integer types.
 
 For any OpenCL C language built-in functions that are mapped onto their GLSL
 4.5 built-in equivalents, the precision requirements of the OpenCL C language
-built-ins are not necessarily honoured.
+built-ins are not necessarily honoured. In general, implementations should
```

Comment:
The "should" here is confusing.

Is this expressing an obligation. e.g. on the user to find and use a Vulkan implementation meeting those constraints?

Or is this an expression of  an expectation likely to be met:  Clspv authors expect Vulkan implementations will satisfy the constraints.

Or is this a promise:  Clspv authors are promising that with a conformant Vulkan implementation, the builtins will satisfy the constraints.  (Or, maybe it's only certain tested Vulkan implementations..?)

Reply:
Clarified this and added a section at the end for known conformance issues. I've encountered only 3 total errors across 5 implementations aiming for relaxed math requirements.